src/IQFeed.CSharpApiClient.Tests.Integration/Lookup/Chains/ChainsFacadeTests.cs

```diff
@@ -68,9 +68,10 @@ public async Task Should_Return_EquityIndexOptions_When_ReqChainIndexEquityOptio
         }
 
         [Test, MaxTime(TimeoutMs)]
-        public void Should_Throw_IQFeedException_When_Chains_Getting_Error()
+        public async Task Should_ReturnNull_When_ReqChainIndexEquityOptionAsync_InvalidSymbol()
         {
-            var ex = Assert.ThrowsAsync<NoDataIQFeedException>(async () => await _lookupClient.Chains.ReqChainIndexEquityOptionAsync("INVALID_SYMBOL_NAME", OptionSideFilterType.CP, string.Empty, 4));
+            var equityIndexOptions = await _lookupClient.Chains.ReqChainIndexEquityOptionAsync("INVALID_SYMBOL_NAME", OptionSideFilterType.CP, string.Empty, 4);
```

Comment:
this one was failing too

Reply:
can you please run these tests on your machine and let me know?
I fixed this one by adding a check and throwing the expected exception, but I'm not sure if that's the right choice
this throws NullReferenceException otherwise because the call tries to access Chains on null

Reply:
it works now